mmv1/templates/terraform/encoders/bigquery_data_transfer.go.tmpl

```diff
@@ -40,6 +40,13 @@ for _, sp := range sensitiveParams {
 	}
 }
 
+for _, sp := range sensitiveWoParams {
+	if auth, _ := d.GetRawConfigAt(cty.GetAttrPath("sensitive_params").IndexInt(0).GetAttr(sp)); !auth.IsNull() {
+		sp = sp[:len(sp)-3] // removes _wo from the end of the param name
```

Comment:
Is it a convention that these should always be suffixed "_wo"?  Can we communicate it more in the YAML comment or somewhere so people know the name takes part in this logic?

Reply:
Rest of the PR LGTM. Wanted to see if you have ideas to make the `sp = sp[:len(sp)-3]` part more future-proof.

Reply:
yeah it's convention to mark write-only attributes with the `_wo` suffix. I can include this as a comment.

Reply:
A comment would help, thanks.